src/telemetry/ProcessStats.cc

```diff
@@ -0,0 +1,200 @@
+#include "zeek/telemetry/ProcessStats.h"
+
+#include "zeek/util.h"
+
```

Comment:
Just curious: is this (very nice) code from an online source or did you piece this together yourself? Might help to add some pointers if the former.

Reply:
This came mostly out of Broker/CAF. See caf/libcaf_core/src/telemetry/importer/process.cpp.